blog.tsx

```diff
@@ -192,14 +192,19 @@ async function loadContent(blogDirectory: string, isDev: boolean) {
   // Read posts from the current directory and store them in memory.
   const postsDirectory = join(blogDirectory, "posts");
 
+  const loadingPromises: Promise<void>[] = [];
+
   // TODO(@satyarohith): not efficient for large number of posts.
   for await (
     const entry of walk(postsDirectory)
   ) {
     if (entry.isFile && entry.path.endsWith(".md")) {
-      await loadPost(postsDirectory, entry.path);
+      // Switch to async to speed larger blogs
+      // await loadPost(postsDirectory, entry.path);
```

Comment:
not sure i understand this comment

Reply:
Agreed. It didn't make sense of the context of the other stub comment above it, so I removed them both since the change resolves the stub and the additional comment brought no new meaning.